tests/src/Kernel/Entity/FieldAccessTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\FieldAccessTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Core\Session\AnonymousUserSession;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+use Drupal\Component\Utility\Unicode;
+use Drupal\node\Entity\NodeType;
+use Drupal\user\Entity\User;
+use Drupal\user\Entity\Role;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Tests OG specific field access.
+ *
+ * @group og
+ */
+class FieldAccessTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['system', 'user', 'field', 'entity_reference', 'node', 'og'];
+
+  /**
+   * @var string
+   *
+   * The machine name of the group node type.
+   */
+  protected $groupBundle;
+
+  /**
+   * @var \Drupal\user\Entity\User
+   */
+  protected $adminUser;
+
+  /**
+   * @var \Drupal\user\Entity\User
+   */
+  protected $authenticatedUser;
+
+  /**
+   * @var \Drupal\Core\Entity\EntityAccessControlHandlerInterface
+   */
+  protected $userAccessControlHandler;
+
+  /**
+   * @var \Drupal\field\Entity\FieldConfig
+   */
+  protected $fieldDefinition;
```

Comment:
This is the field definition of the OG audience field, maybe add a line of documentation explaining this.